packages/resin-event-log/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+* Convert to JS
```

Comment:
Right, ES6 or TS, with compilation to ES5 for the time being but this isn't either of those